cypress/e2e/modals.cy.ts

```diff
@@ -1,10 +1,11 @@
-import MaputnikDriver from "./maputnik-driver";
+import { MaputnikDriver } from "./maputnik-driver";
 
 describe("modals", () => {
-  let { beforeAndAfter, when, get, should } = new MaputnikDriver();
+  let { beforeAndAfter, when, get, then } = new MaputnikDriver();
   beforeAndAfter();
+
   beforeEach(() => {
-    when.setStyle("");
+    when.visit("/");
```

Comment:
This isn't exactly the same as the original test...

Reply:
We should probably rename setStyle... next PR